alerting/src/test/kotlin/org/opensearch/alerting/DocumentMonitorRunnerIT.kt

```diff
@@ -22,50 +22,6 @@ import java.time.temporal.ChronoUnit.MILLIS
 
 class DocumentMonitorRunnerIT : AlertingRestTestCase() {
 
-    fun `test execute monitor with dryrun`() {
```

Comment:
There is no reason to delete this. Maybe rename it to `test execute monitor with dryrun and single inputed index`